salt/modules/bigip.py

```diff
@@ -0,0 +1,1723 @@
+# -*- coding: utf-8 -*-
+'''
+An execution module which can manipulate an f5 bigip via iControl REST
+    :maturity:      develop
+    :platform:      f5_bigip_11.6
+'''
+
+# Import python libs
+import requests
```

Comment:
I dont' think it has sessions yet, we need to get that in, I think it only has query still.

I am going to merge this in, we can fix the imports error and deprecate requests out of here later, I don't want this code to go stale just because of a little thing like this!